src/components/BrowseInput/BrowseInput.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import * as styles from './BrowseInput.sass';
+import IReactComponentProps from '../../common/structures/IReactComponentProps';
+
+// const remote = require('electron').remote;
```

Comment:
These code blocks need to be uncommented. Did you do this for Styleguidist? If so, we can add a conditional to test for Electron.

Reply:
Good catch. I just missed these. There was an issue originally, but I think it's all been sorted out. Otherwise, like you mentioned, I'll add a conditional.